index-es.html

```diff
@@ -0,0 +1,104 @@
+  font-size: 18px;
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
+}
+
+.footer {
+  padding: 40px 0;
+  background: #efefef;
+  margin: 0;
+  margin-top: 40px;
+  max-width: 100%;
+}
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+<h1>C&oacute;digo de Conducta de la comunidad</h1>
+
+<p>Se requiere que todos los miembros acepten el siguiente C&oacute;digo de Conducta. Se espera de todos los miembros de la comunidad que hagan cumplir este C&oacute;digo en todos los eventos con ayuda de los organizadores. Esperamos cooperaci&oacute;n de todos los participantes de la comunidad para ayudar a fomentar un ambiente seguro para todos.</p>
+
+<h2>¿Necesitas ayuda?</h2>
+
+<p>Los datos de contacto de los organizadores se encuentran en nuestra web y en cualquier correo electr&oacute;nico que hemos enviado.</p>
+
+<h2>La versi&oacute;n corta</h2>
+
+<p>Nuestra comunidad est&aacute; dedicada a proporcionar una experiencia libre de acoso para todos, independientemente del g&eacute;nero, de la identidad y la expresi&oacute;n de g&eacute;nero, edad, orientaci&oacute;n sexual, discapacidad, apariencia f&iacute;sica, tama&ntilde;o corporal, raza, etnia, religi&oacute;n (o falta de ella), o preferencias tecnol&oacute;gicas. No toleramos el acoso a miembros de la comunidad cualquiera que sea su forma. El lenguaje y las met&aacute;foras sexuales no son apropiados en ning&uacute;n evento de la comunidad, incluyendo ponencias, talleres, fiestas, Twitter y otros medios online. Los participantes que violen estas reglas pueden ser sancionados o expulsados del evento y de cualquier evento futuro <em>sin un reembolso</em> (si aplica), o de la propia comunidad, a discreci&oacute;n de los organizadores.</p>
+
+<h2>La versi&oacute;n menos corta</h2>
+
+<p>Acoso incluye comentarios verbales ofensivos relacionados con g&eacute;nero, identidad y expresi&oacute;n de g&eacute;nero, edad, orientaci&oacute;n sexual, discapacidad, apariencia f&iacute;sica, tama&ntilde;o corporal, raza, etnia, religion, preferencias tecnol&oacute;gicas, im&aacute;genes de &iacute;ndole sexual en espacios p&uacute;blicos, intimidaci&oacute;n intencionada, acechar, seguir, toma de fotograf&iacute;as o grabaci&oacute;n de video de una manera hostigadora, interrupci&oacute;n continuada de ponencias u otros eventos, contacto f&iacute;sico inapropiado y proposiciones sexuales no deseadas.</p>
+
+<p>Se espera que aquellos participantes a los que se les solicite cesar cualquier tipo de acoso lo hagan de manera inmediata.</p>
+
+<p>Los esp&oacute;nsors tambi&eacute;n est&aacute; sujetos a la pol&iacute;tica anti acoso. En particular, los esp&oacute;nsors no deber&aacute; utilizar im&aacute;genes, movimientos u otro material sexualizado. Los anfitriones y asistentes, incluyendo a los esp&oacute;nsors, empleados y voluntarios, no deber&aacute;n utilizar ropa/uniformes/disfraces sexualizados o crear un ambiente sexualizado de cualquier otra manera.</p>
```

Comment:
Good transalation. 
Only one question. "imagenes, movimientos u otro material sexualizado"  Movimientos sounds a bit strange here, why not use "actividades" ?